application.json.template

```diff
@@ -1,7 +1,14 @@
 {
   "server": {
     "security": {
-      "enable": false
+      "enable": false,
+      "npmModule": {
+        "name": "basic-security-client"
+      },
+      "configModule": {
+        "user": "admin",
```

Comment:
environment vars are easy to use:

"user": "${AUTH_USER}"
"password": ${AUTH_PASSWORD}

This must be in the documentation.

Reply:
Fixed...